polyglot/piranha/README.md

```diff
@@ -469,6 +469,10 @@ The purpose of Piranha Arguments is determining the behavior of Piranha.
 
 ## Contributing
 
+Prerequisites: 
+* Install [pre-commit](https://pre-commit.com/)
```

Comment:
So, this requires manual installation steps? Isn't there a way to add the git pre-commit hook automatically on any local build like we do using Gradle in Java projects? I think I'd rather have a checked in pre-commit-hook.template and copy it on build, even if the configuration is less clean, just to avoid more manual installation steps.